sklearn/neural_network/tests/test_mlp.py

```diff
@@ -437,16 +449,38 @@ def test_predict_proba_multi():
     assert_array_equal(y_log_proba, np.log(y_proba))
 
 
+def test_predict_proba_multilabel():
+    # Test that predict_proba works as expected for multilabel.
+    # Multilabel should not use softmax which makes probabilities sum to 1
+    X, y = make_multilabel_classification(n_samples=50, random_state=0,
+                                          return_indicator=True)
+
+    clf = MLPClassifier(hidden_layer_sizes=5)
+    clf.fit(X, y)
```

Comment:
Please use the LBGFS solver to avoid the convergencewarning on such a small dataset.